recipes/netl-ap-map-flow/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set repo_name = "netl-ap-map-flow" %}
+{% set repo_base_url = "https://github.com/stadelmanma/netl-ap-map-flow" %}
+{% set module_name = "ApertureMapModelTools" %}
+{% set version = "0.0.2" %}
+{% set sha256 = "0c571ac190fe0a69aa94a654f37dc778b487704d4b491144b875e500adc296a6" %}
+{% set py_mod_deps = ['numpy x.x', 'scipy', 'pillow >=3.4.0', 'pyyaml']%}
+
+package:
+  name: {{ repo_name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ repo_name }}-{{ version }}.tar.gz
+  url: {{ repo_base_url }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py2k]
+  skip: True  # [py<34]
+  skip: True  # [linux32]
+  skip: True  # [win32]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+    - libtool  # [not win]
+    - gcc  # [not win]
+    - libgfortran  # [not win]
+    - m2w64-gcc-fortran  # [win]
+    {% for each_py_mod_dep in py_mod_deps %}
+    - {{ each_py_mod_dep }}
+    {% endfor %}
```

Comment:
Can you add these explicitly like the others without the loop? conda-forge's maintenance scripts won't be able to function properly with this.